### PR TITLE
User Docs 0.7 updates & improvements

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary

Corrections, new screenshots and several improvements.

### Reviewer guidance

New [content for coaches](https://github.com/learningequality/kolibri/pull/3039/files#diff-f95adbc56f34cee5a14e00832dd61365) might need a read-through... 😅 

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [ ] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by radinamatic** [comment]
@indirectlylit I just now realize this is targeted on `develop` instead of `release-v0.7` branch, but it should ideally update the 0.7 version of the docs.

@jeepurs This latest update should cover practically everything currently available in Kolibri (plus a couple of `Coming soon!` items)  from your last year's prep docs. 🙂

**Comment by ivanistheone** [comment]
Here is a link to a "red-blue diff" which might be a bit easier to see the changes.

https://minireference.com/static/tmp/User_Docs_0.7_updates_n_improvements.html

Red = text removed

Blue = text added

**Comment by indirectlylit** [comment]
> @indirectlylit I just now realize this is targeted on develop instead of release-v0.7 branch, but it should ideally update the 0.7 version of the docs.

Sounds right. I'll try switching the target. We may need to do some rebase or cherry-pick work...

**Comment by indirectlylit** [comment]
yup - that totally broke it :) let me know if you need help retargetting

File: docs/access.rst [hunk 1]
```diff
@@ -81,14 +83,17 @@ While **Kolibri** is up and running on the device where you installed it, other
...
 
-      Find out the IP/Server URL in the Device > Info tab.
+    Find out the IP/Server URL in the Device > Info tab.
 
 .. note::
   You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find out the IP address of the device running the **Kolibri**.
 
+.. tip::
+  You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find out the IP address of the device running the **Kolibri**.
+
```
Comment (indirectlylit) [inline docs/access.rst:96]: The "tip" duplicates the "note" above it

**Comment by radinamatic** [comment]
@jeepurs 

I've uploaded my [local build to our shared folder](https://drive.google.com/open?id=1tETJscbScJUtZZIFysGPdZFv6_eJLi3P). Just download, unzip and open the `index.html` in your browser. 

It will be much easier to read, but not so easy to commend and request changes...

File: docs/access.rst (hunk 1 above)
Comment (radinamatic) [reply docs/access.rst:96]: Forgot to delete... 😒

**Comment by jeepurs** [comment]
@radinamatic I breezed through the Exercise progress (under the Coach section) and the FAQ section.  If there are other sections you want me to fine-tooth comb, let me know.  Otherwise, looks awesome!

**Comment by radinamatic** [comment]
Thanks @jeepurs, that should be Ok! 👍 
There are a lot of other small changes, but mostly corrections, rewording and reshuffling stuff around trying to find the best possible overall structure!

**Comment by indirectlylit** [comment]
@radinamatic your work in `_get_support.rst` conflicts with some work that @benjaoming did in #2990

File: docs/access.rst (hunk 1 above)
Comment (benjaoming) [reply docs/access.rst:96]: In the longer run, I don't think this should be a user documentation, firstly it requires command line, secondly this information is displayed when running kolibri from command line or `kolibri status`, and any UI should mention it as well.

**Comment by benjaoming** [comment]
This is not so nice to back-port, but am working on it..

**Comment by benjaoming** [comment]
Closing in favor of #3087 which is targeted for release-v0.7.x and resolves a bunch of conflicts (hopefully the right way)

**Pull request closed without merging**